client/my-sites/site-settings/disconnect-site/index.jsx

```diff
@@ -9,49 +9,40 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
-import DisconnectSurvey from './disconnect-survey';
-import DocumentHead from 'components/data/document-head';
-import FormattedHeader from 'components/formatted-header';
-import Main from 'components/main';
-import NavigationLink from 'components/wizard/navigation-link';
+import { DownFlow } from './down-flow';
```

Comment:
Nit: it's more common to export React components as default exports, especially if there's only one in the file.

One practical benefit of that is that libraries that do lazy loading of React components usually assume a `default` export by default.

Reply:
done in e91541c3e89649838c4a0bc766308f0a63ecec2f